components/feature/search/src/main/assets/extensions/search/manifest.template.json

```diff
@@ -0,0 +1,220 @@
+{
+  "manifest_version": 2,
+  "applications": {
+    "gecko": {
+      "id": "search_telemetry@mozac.org"
```

Comment:
🔥 Let's not change the ID relative to https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/assets/extensions/ads/manifest.template.json#L5.

With this we end up with two installed extensions and we would need migration code to uninstall the old one first!

Reply:
Let's just keep the ID the same and make sure we remove the extension from Fenix.

Reply:
(Likely also worth retesting, because with this you had the old extension still installed as well, unless you tested on a new profile).

Reply:
[Here](https://debug-ping-preview.firebaseapp.com/pings/petruNewSearch2) is a metrics ping from Fenix (Jul 8, 2021, 14:48:28) which shows telemetry for searches and an ad being clicked.
This was recorded after with the old id used for extensions on a fresh install of the app.